Xamarin.Forms.Core/Button.cs

```diff
@@ -31,32 +39,42 @@ public class Button : View, IFontElement, ITextElement, IButtonController, IElem
 
 		public static readonly BindableProperty FontAttributesProperty = FontElement.FontAttributesProperty;
 
-		public static readonly BindableProperty BorderWidthProperty = BindableProperty.Create("BorderWidth", typeof(double), typeof(Button), -1d);
+		public static readonly BindableProperty BorderWidthProperty = BindableProperty.Create("BorderWidth", typeof(double), typeof(Button), DefaultBorderWidth);
```

Comment:
I don't see the need to introduce a level of indirection here. one can check the default value by doing `BorderWidthProperty.GetDefaultValue()`